R-package/README.md

```diff
@@ -31,7 +31,7 @@ You can perform installation either with **Apple Clang** or **gcc**. In case you
 export CXX=/usr/local/bin/g++-8 CC=/usr/local/bin/gcc-8
 ```
 
-### Install
+### Install <a name="install-source"></a>
```

Comment:
I don't think it's needed. GitHub, pkgdown, sphinx, RTD automatically generate URL anchors for headings. Also, this change breaks backward compatibility: old links which point to `#install` are broken now.

Reply:
ah! yes you are right, I will fix that.

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/2664/commits/822cd15c175167407b4aa25f4a1cf2e30cb1e21f